src/activity_handlers.cpp

```diff
@@ -56,6 +56,29 @@ void activity_handlers::burrow_finish(player_activity *act, player *p)
     g->m.destroy(dirx, diry, true);
 }
 
+bool butcher_cbm_item( const std::string item, const point &pos, int age )
```

Comment:
If you're going to take the point as reference, why not the string as well? And you could make the age const.
Please rewrite the if condition to `one_in( 2 )`, that is far easier to understand.